src/test/java/LoginTest.java

```diff
@@ -21,6 +21,7 @@ public void login() {
     @Test
     void loginTest() {
         Assertions.assertEquals(LOGIN, mailPage.getAccountName(), "Login failed");
+        takeScreenshot("user_logged.png");
```

Comment:
put this before the assertion. If test fails on assertions you will be able to make a screenshot